packages/ic-http-certification-tests/tests/v2_response_verification_certification_scenarios.rs

```diff
@@ -408,15 +410,17 @@ mod tests {
             ),
         );
 
+        let mut expected_headers = expected_response
+            .headers
+            .iter()
+            .map(|(key, value)| (key.to_lowercase(), String::from(value)))
```

Comment:
nit: as we don't filter anymore, `to_lowercase()` seems obsolete here. Could we just clone() `expected_response.headers` rather than iterating through them?

Reply:
We still need the headers to be lowercase as the "verified" response will have all the headers in lowercase. The filtering still happens during the verification.